README.md

```diff
@@ -35,9 +35,10 @@ Thus, you can add an rss/xml feed to share stories and experiences to be present
 and it will show up in the feed for the respective repository and the [aggregated blogs feed](https://hpc.social/blog/)
 here. Here is how to get started, for either of the above!
 
-1. Fork the repository that you've chosen in the step above (**you should not be forking this repository where you are reading!**), clone it to your machine, and checkout a new branch.
+1. Fork the repository that you've chosen in the step above (**you should not be forking this repository where you are reading!**), checkout a new branch, and clone it to your machine.
 1. Add your entry to the [_data/authors.yml](_data/authors.yml) file
 1. Generate your set of posts (instructions below)
+1. Push your change to GitHub
```

Comment:
@mvanwaveren this line would be ok to add, if you want to update the PR to change the first set of changes (or just undo them) and keep this line, I think it would be a good addition. Feel free to use any of the text I provided about the process if you think there isn't enough detail.